ogc-example/src/main/scala/geotrellis/server/ogc/wms/WmsView.scala

```diff
@@ -18,31 +18,59 @@ package geotrellis.server.ogc.wms
 
 import geotrellis.server._
 import geotrellis.server.ogc._
+import geotrellis.server.ogc.utils._
 import geotrellis.server.ogc.params.ParamError
 import geotrellis.server.ogc.wms.WmsParams.{GetCapabilities, GetMap}
 
 import geotrellis.raster.RasterExtent
 import geotrellis.raster._
 import com.azavea.maml.error._
 import com.azavea.maml.eval._
-import scalaxb.CanWriteXML
 import org.http4s.scalaxml._
 import org.http4s.circe._
 import org.http4s._
 import org.http4s.dsl.io._
-import org.http4s.implicits._
 import _root_.io.circe.syntax._
-import cats._
 import cats.implicits._
 import cats.effect._
 import cats.data.Validated._
 import com.github.blemale.scaffeine.{Cache, Scaffeine}
+import org.log4s.getLogger
+import scalaxb.DataRecord
 
 import java.net.URL
+
 import scala.concurrent.duration._
+import scala.xml.Elem
 
 class WmsView(wmsModel: WmsModel, serviceUrl: URL) {
-  val logger = org.log4s.getLogger
+  val logger = getLogger
+
+  val extendedCapabilities: List[DataRecord[Elem]] = {
```

Comment:
All the extended parameters moved into the user codebase as well.

Reply:
What do you mean? `WmsView` is not use code

Reply:
@echeipesh I thought that everything that is in `ogc-examples` project is a user code.